tools/ReleaseTools.psm1

```diff
@@ -332,28 +345,25 @@ function New-DraftRelease {
         [ValidateSet([RepoNames])]
         [string]$RepositoryName
     )
+    param(
+        [Parameter(ValueFromPipeline)]
+        [string[]]$Assets
+    )
     $Version = Get-Version -RepositoryName $RepositoryName
     $Changelog = (Get-FirstChangelog -RepositoryName $RepositoryName) -join "`n"
     $ReleaseParams = @{
```

Comment:
I notice all the local variable names here are in PascalCase, but usually for local variables I'd use camelCase

Reply:
🤷